Module3/Program.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using DeliveryService.Data;
+
+namespace DeliveryService
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var data = new StoreContext();
+            Console.WriteLine("-----Task1-----");
+            Task1(data);
+            Console.WriteLine("-----Task2-----");
+            Task2(data);
+            Console.WriteLine("-----Task3-----");
+            Task3(data);
+            Console.WriteLine("-----Task4-----");
+            Task4(data);
+            Console.WriteLine("-----Task5-----");
+            Task5(data);
+        }
+
+        static void Task1(StoreContext data)
+        {
+            var products = data.Products.OrderBy(p => p.Name).ToList();
+            foreach(var product in products)
+            {
+                Console.WriteLine($"{product.Name}");
+            }
+        }
+
+        static void Task2(StoreContext data)
+        {
+            var products = data.Products.Join(data.Users,
+                p => p.SellerId,
+                u => u.Id,
+                (p, u) => new { SellerName = u.Name, ProductName = p.Name });
+            Console.WriteLine("Seller Name | Product Name");
+            foreach(var product in products)
+            {
+                Console.WriteLine($"{product.SellerName}     |     {product.ProductName}");
+            }
+        }
+
+        static void Task3(StoreContext data)
+        {
+            var categories = data.Categories;
+            var products = data.Products;
+            Console.WriteLine("Category Name | Products Count");
+            foreach (var category in categories)
+            {
+                var productsInCategory = products.Where(x => x.CategoryId == category.Id).ToList();
+                Console.WriteLine($"{category.Name}     |     {productsInCategory.Count}");
```

Comment:
please make it via LINQ

Reply:
Didn't i do it via LINQ? Or you mean i should use Join?

Reply:
Remade it using Join.

Reply:
great!